overlord/snapstate/snapstate.go

```diff
@@ -2089,20 +2100,48 @@ func removeInactiveRevision(st *state.State, name, snapID string, revision snap.
 func RemoveMany(st *state.State, names []string) ([]string, []*state.TaskSet, error) {
 	removed := make([]string, 0, len(names))
 	tasksets := make([]*state.TaskSet, 0, len(names))
+
+	var totalSnapshotsSize uint64
+	path := dirs.SnapdStateDir(dirs.GlobalRootDir)
+
 	for _, name := range names {
-		ts, err := Remove(st, name, snap.R(0), nil)
+		ts, snapshotSize, err := removeTasks(st, name, snap.R(0), nil)
 		// FIXME: is this expected behavior?
 		if _, ok := err.(*snap.NotInstalledError); ok {
 			continue
 		}
+		if _, ok := err.(*InsufficientSpaceError); ok {
```

Comment:
We don't need this anymore, right? removeTasks will no longer return this (?)

Reply:
Ah, of course, thanks for catching!

Reply:
Removed.